ReturnBookControl.java

```diff
@@ -1,31 +1,31 @@
 public class ReturnBookControl {
 
 	private ReturnBookUI returnBookUi;
-	private enum RetunrBookControlState { INITIALISED, READY, INSPECTING };
-	private RetunrBookControlState state;
+	private enum ReturnBookControlState { INITIALISED, READY, INSPECTING };
+	private ReturnBookControlState state;
 	
 	private library library;

```

Comment:
library1

Reply:
library library1 is a similar format to past code  please change so that its consistent  throughout the file

Reply:
Alright. Done